<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>735</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>For testing taps</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>541</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <widget class="QCheckBox" name="singleCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Single</string>
        </property>
       </widget>
       <widget class="QLabel" name="singleDirection">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>60</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>62</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="singleLCD">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="singleTime">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>10</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="lastSingle">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>10</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>(last single)</string>
        </property>
       </widget>
       <widget class="QLabel" name="singlePcs">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>10</y>
          <width>21</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>pcs</string>
        </property>
       </widget>
       <widget class="QLabel" name="singleFrequency">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>70</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string/>
       </property>
       <widget class="QCheckBox" name="doubleCheckBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>71</width>
          <height>18</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>18</height>
         </size>
        </property>
        <property name="text">
         <string>Double</string>
        </property>
       </widget>
       <widget class="QLabel" name="doubleDirection">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>80</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>62</width>
          <height>17</height>
         </size>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLCDNumber" name="doubleLCD">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>71</width>
          <height>51</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="doubleTime">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>30</y>
          <width>62</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="lastDouble">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>81</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>(last double)</string>
        </property>
       </widget>
       <widget class="QLabel" name="doublePcs">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>30</y>
          <width>21</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>pcs</string>
        </property>
       </widget>
       <widget class="QLabel" name="doubleFrequency">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>80</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
